terraform/team-members-access/infra-deploy-playground.tf

```diff
@@ -30,6 +30,26 @@ resource "aws_iam_group_policy" "infra_deploy_playground" {
           "arn:aws:ssm:us-west-1:890664054962:parameter/staging/ansible/all/*",
         ]
       },
+      {
+        "Effect" : "Allow",
+        "Action" : "ec2-instance-connect:SendSSHPublicKey",
+        "Resource" : [
+          "arn:aws:ec2:us-west-1:890664054962:instance/i-04db6ae41566bd1d9",
+        ],
+        "Condition" : {
+          "StringEquals" : {
+            "ec2:osuser" : "shepmaster"
```

Comment:
I'm not sure about this value; the docs suggest it being the user that the AMI creates (e.g. `ubuntu` for us, I think).

If it is *my* OS username, that'd be `shep`.